extension/packages/nextjs/components/Header.tsx.args.mjs

```diff
@@ -0,0 +1,6 @@
+export const menuIconImports = `import { EyeIcon } from "@heroicons/react/24/outline";`;
+export const menuObjects = `{
+  label: "EIP-712",
+  href: "/eip-712",
+  icon: <EyeIcon className="h-4 w-4" />,
```

Comment:
nit: Should we use here `DocumentCheckIcon` ? 

![Screenshot 2024-06-13 at 11 47 02 AM](https://github.com/scaffold-eth/create-eth-extensions/assets/80153681/c0065627-afef-4521-af2c-f4c2091020a3)

Reply:
I used EyeIcon because user sees the data he signs. But not in our burner-wallet 🤷‍♂️ . Updated [6bd0c01](https://github.com/scaffold-eth/create-eth-extensions/pull/3/commits/6bd0c01eaecdd1b8ae05c29897ee471f9e25d1b3)